src/main/java/pl/edu/mimuw/model/argumentless/ArgumentlessExpression.java

```diff
@@ -0,0 +1,24 @@
+package pl.edu.mimuw.model.argumentless;
+
+import pl.edu.mimuw.model.Expression;
+
+public abstract class ArgumentlessExpression implements Expression {
+  public abstract double evaluateAtPoint(double point);
```

Comment:
to jest niepotrzebne w klasie abstakcyjnej, jak klasa by dodawała nowe to należałoby je tu zdefiniować, ale tych z interfejsu nie ma potrzeby powtarzać